components/ActionSheet/src/private/MDCActionSheetListViewController.m

```diff
@@ -0,0 +1,157 @@
+/*
+ Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "MDCActionSheetListViewController.h"
+#import "MDCActionSheetItemView.h"
+#import "../MDCActionSheetController.h"
+#import "MaterialTypography.h"
+
+NSString *const kReuseIdentifier = @"BaseCell";
+static const CGFloat kActionItemLabelPadding = 18.f;
+static const CGFloat kActionItemTrailingPadding = 16.f;
+
+@interface MDCActionSheetListViewController () <UITableViewDataSource>
+@end
+
+@implementation MDCActionSheetListViewController {
+  BOOL _mdc_adjustsFontForContentSizeCategory;
```

Comment:
You can use `@synthesize mdc_adjustsFontForContentSizeCategory = _ mdc_adjustsFontForContentSizeCategory;` instead of having to create an ivar and getter method.

Reply:
Done